### ISSUE TITLE
Ring Banning IP

### ISSUE THREAD

**Issue by typkrft** [dgreif/ring#265]
# Bug Report

### Describe the Bug
I believe my public IP is getting banned by Ring.com. When I go to the website on any device using my wifi network, after setting up the plugin, it says Invalid Username or Password. In the JS Console it's logging a 403 error when I hit submit. If I log in to the app or the website via a cellular connection on the same affected devices, I log in without issue.  If restart my router to get a new public IP I am then able to log into the website again, until I try to use homebridge-ring again. I reproduced the issue 3 times now.

### To Reproduce

Steps taken within homebridge-config-ui-x v4.12.0
Install homebridge-ring 
Set credentials
Restart Homebridge 
RIng camera's dont show up.
Try logging into the website or using ring-auth-cli no longer work on my network. 

### Expected behavior
 A clear and concise description of what you expected to happen.

### Screenshots/Logs
 If applicable, add screenshots and log output to help explain your problem.

### Additional context
 Add any other context about the problem here.

### Homebridge Ring Config
Post homebridge-ring platform config  _without sensitive information_
```

{
            "email": "d***g@****.com",
            "password": "********",
            "refreshToken": "e*****d**",
            "platform": "Ring"
        }

```

### Environment

 - OS: macOS 10.15
 - Node.js: 12.4.0
 - NPM: 6.14.2
 - homebridge-ring: v7.2.0 (2020-02-26)
 - homebridge: v0.4.46

**Comment by dgreif** [comment]
@typkrft that's really unfortunate that they are blocking your IP address.  I'm not sure if there is much I can do from the plugin side, other than trying to very closely match all of the user agent and device types that they use from the app.  Do they seem to block the IP _immediately_, or is it after some amount of time?

**Comment by typkrft** [comment]
It’s within 15 seconds for sure. I’ve noticed after I install the plugin and restart homebridge a few of my other plugins stop functioning. I’m going to try and add the configuration manually and remove all other plugins to see if that has any effect. The reason I mention that is I’m wondering if Homebridge is restarting and/or trying to Start the plugin rapidly and I’m DDoSing them.

Have an outstanding day,

Brandon Shockley
SCAC: SRRV
Mobile: 765.480.4113
Fax: 480.718.7585

**Comment by dgreif** [comment]
@typkrft did you ever figure out if this was an issue on your end?

**Comment by typkrft** [comment]
Honestly, I was never able to isolate the cause. I still think another module or homebridge possibly was causing the plugin to rapidly reload and login maybe. I made some massive changes to my network infrastructure (unrelated), removed homebridge and started from scratch. I installed it as the first plug in and have not been banned since. Thank you for your work on the plugin by the way and sorry for not reporting back sooner.

Have an outstanding day,

Brandon Shockley
SCAC: SRRV
Mobile: 765.480.4113
Fax: 480.718.7585
